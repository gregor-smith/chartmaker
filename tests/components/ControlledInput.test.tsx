test.each([ 'text', 'password' ] as const)('renders input', _type => {

})


test.todo('change event calls onChange prop')
