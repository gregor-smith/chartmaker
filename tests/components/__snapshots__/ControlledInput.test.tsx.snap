// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders disabled prop 1`] = `
.emotion-0 {
  background: black;
  color: white;
  border: 1px solid white;
  padding: 0.25rem;
  width: 100%;
}

<div
  id="test-render-container"
>
  <input
    class="emotion-0"
    disabled=""
    id="test-input"
    type="search"
    value="test value"
  />
</div>
`;

exports[`renders disabled prop 2`] = `
.emotion-0 {
  background: black;
  color: white;
  border: 1px solid white;
  padding: 0.25rem;
  width: 100%;
}

<div
  id="test-render-container"
>
  <input
    class="emotion-0"
    id="test-input"
    type="search"
    value="test value"
  />
</div>
`;

exports[`renders disabled prop 3`] = `
.emotion-0 {
  background: black;
  color: white;
  border: 1px solid white;
  padding: 0.25rem;
  width: 100%;
}

<div
  id="test-render-container"
>
  <input
    class="emotion-0"
    id="test-input"
    type="search"
    value="test value"
  />
</div>
`;

exports[`renders text type input 1`] = `
.emotion-0 {
  background: black;
  color: white;
  border: 1px solid white;
  padding: 0.25rem;
  width: 100%;
}

<div
  id="test-render-container"
>
  <input
    class="emotion-0"
    id="test-input"
    type="search"
    value="test value"
  />
</div>
`;

exports[`renders text type input 2`] = `
.emotion-0 {
  background: black;
  color: white;
  border: 1px solid white;
  padding: 0.25rem;
  width: 100%;
}

<div
  id="test-render-container"
>
  <input
    class="emotion-0"
    id="test-input"
    type="password"
    value="test value"
  />
</div>
`;
